client/components/TestRun/index.jsx

```diff
@@ -389,7 +389,7 @@ class TestRun extends Component {
         );
     }
 
-    renderTest({ run, test, git_hash, at_key, testIndex, testerId }) {
+    renderTest({ run, test, git_hash, at_key, testIndex, testerId }, heading) {
```

Comment:
@dawnwages I didn't like the idea of having a global heading variable while the component takes on lifecycle methods, so I just passed the heading as a param to renderTest(). It's not perfect, but it takes care of the global var code smell.